cmd/restic/integration_test.go

```diff
@@ -799,6 +799,25 @@ func TestCopyIncremental(t *testing.T) {
 		len(copiedSnapshotIDs), len(snapshotIDs))
 }
 
+func TestCopyUnstableJSON(t *testing.T) {
+	env, cleanup := withTestEnvironment(t)
+	defer cleanup()
+	env2, cleanup2 := withTestEnvironment(t)
+	defer cleanup2()
+
+	// contains a symlink created using `ln -s '../i/'$'\355\246\361''d/samba' invalid-symlink`
```

Comment:
Nitpick, but this is not an *invalid* symlink, as I said over at #3267.

Reply:
I've changed the comment to say "broken-symlink". The `copy-unstable-json.tar.gz` still contains the filename `invalid-symlink`.